src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage.mdx

```diff
@@ -171,5 +202,383 @@ let poolSizeSet = NewRelic.setMaxOfflineStorageSize(1000)
 
         </TabsPageItem>
 
+
+ <TabsPageItem id="capacitor">
+
+ <Callout variant="important">
+Offline storage is enabled by default but can be disabled with a feature flag using the agent configuration.
```

Comment:
@ndesai-newrelic Can you add the code sample of how to disable this (similar to the iOS/Android callouts)?